scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,300 @@
+type: function
+desc: "Append a memory copy command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendMembufferCopyExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object."
+    - type: $x_mem_handle_t
+      name: hSrcMem
+      desc: "[in] The data to be copied."
+    - type: $x_mem_handle_t
+      name: hDstMem
+      desc: "[in] The location the data will be copied to."
+    - type: "size_t"
+      name: srcOffset
+      desc: "[in] Offset into the source memory."
+    - type: "size_t"
+      name: dstOffset
+      desc: "[in] Offset into the destination memory"
+    - type: "size_t"
+      name: size
+      desc: "[in] The number of bytes to be copied."
+    - type: uint32_t
+      name: numSyncPointsInWaitList
+      desc: "[in] The number of sync points in the provided dependency list."
+    - type: "const $x_exp_command_buffer_sync_point_t*"
+      name: pSyncPointWaitList
+      desc: "[in] A list of sync points that this command depends on."
+    - type: "$x_exp_command_buffer_sync_point_t*"
+      name: pSyncPoint
+      desc: "[out] sync point associated with this command"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_EXP
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_WAIT_LIST_EXP
+        - "`pSyncPointWaitList == NULL && numSyncPointsInWaitList > 0`"
+        - "`pSyncPointWaitList != NULL && numSyncPointsInWaitList == 0`"
+    - $X_RESULT_ERROR_INVALID_MEM_OBJECT
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Append a rectangular memory copy command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendMembufferCopyRectExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object."
+    - type: $x_mem_handle_t
+      name: hSrcMem
+      desc: "[in] The data to be copied."
+    - type: $x_mem_handle_t
+      name: hDstMem
+      desc: "[in] The location the data will be copied to."
+    - type: $x_rect_offset_t
+      name: SrcOrigin
+      desc: "[in] Origin for the region of data to be copied from the source."
+    - type: $x_rect_offset_t
+      name: DstOrigin
+      desc: "[in] Origin for the region of data to be copied to in the destination."
+    - type: $x_rect_region_t
+      name: Region
+      desc: "[in] The extents describing the region to be copied."
+    - type: "size_t"
+      name: SrcRowPitch
+      desc: "[in] Row pitch of the source memory."
+    - type: "size_t"
+      name: SrcSlicePitch
+      desc: "[in] Slice pitch of the source memory."
+    - type: "size_t"
+      name: DstRowPitch
+      desc: "[in] Row pitch of the destination memory."
+    - type: "size_t"
+      name: DstSlicePitch
```

Comment:
Some more instances of names that should be camelCase in this `params:` list.

Reply:
These should all be fixed now.